ush/test_retrieve_data.py

```diff
@@ -494,3 +496,59 @@ def test_ufs_lbcs_from_aws(self):
             path = os.path.join(tmp_dir, "*")
             files_on_disk = glob.glob(path)
             self.assertEqual(len(files_on_disk), 2)
+
```

Comment:
The unit tests appear to be failing for the newly added `test_rap_obs_from_hpss` and `test_rap_e_obs_from_hpss` tests due to HPSS not being available.  These tests should pass if:

`@unittest.skipIf(os.environ.get("CI") == "true", "Skipping HPSS tests")`

is added before these two tests.